src/main/resources/application.properties

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 quarkus.swagger-ui.always-include=true
-quarkus.native.additional-build-args=-march=compatibility
+quarkus.native.additional-build-args=-march=compatibility, -H:+UnlockExperimentalVMOptions, -H:ResourceConfigurationFiles=resources-config.json
```

Comment:
question: Is `-H:+UnlockExperimentalVMOptions` needed for what exactly? I was expecting only the `resources-config.json`.